app/javascript/pulfalight/toc.es6

```diff
@@ -59,6 +59,15 @@ export default class TocBuilder {
   }
 
   setupToggleElement() {
+    var prev_collection=localStorage.getItem("collection");
+    var current_collection=$('#document').data('document-id').split("_").at(0);
+
+    // reset toggle state if the user has moved to a new collection
+    if(prev_collection !== current_collection) {
```

Comment:
If I navigate between collections between tabs, is this gonna result in strange behavior?

Reply:
Yes, you're correct @tpendragon. Thanks for catching that. I think I need to store the concatenated value of the the browser tab id and the collection id as elements of an array of current tabs in a LocalStorage variable.

Reply:
Oh wow I didn't even know tab ids were a thing. That's cool.